src/app/shared/markdown-wrapper/markdown-wrapper.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import DOMPurify from 'dompurify';
+import { MarkdownService } from 'ngx-markdown';
+
+@Component({
+  selector: 'markdown-wrapper',
+  templateUrl: './markdown-wrapper.component.html',
+  styleUrls: ['./markdown-wrapper.component.scss'],
+})
+
+/**
+ Wrapper for user-input markdown to provide custom sanitization not accessible through the built-in sanitizer, DOMSanitize.
```

Comment:
This is wrapping ngx-markdown. It is not wrapping the component, but using the service. I feel like the comment could better reflect that. I may be biased because I know what we've been discussing, but I expect a wrapper to wrap another piece of software. :)

Reply:
Updated description.